doc/engineering-reference/src/loop-equipment-sizing-and-other-design-data/component-sizing.tex

```diff
@@ -649,6 +649,52 @@ \subsubsection{Rated Sensible Cooling Capacity}\label{rated-sensible-cooling-cap
   \(\dot{Q}_{fan,heat,des}\) : is the design fan heat (W) - see Section \ref{design-fan-heat}
 \end{itemize}
 
+\subsubsection{Rated Cooling Power Consumption}\label{rated-cooling-power-consumption}
+
+The following calculations are performed to determine the rated cooling power consumption.
+
+The calculation for coil operating temperatures (inlet and outlet) are identical to that done for \emph{Coil:Cooling:Water}. 
+
+\begin{equation}
+  T_{WB,ratio} = \frac{T_{WB,air,in,des}+273.15C}{283.15C}
+\end{equation}
+
+\begin{equation}
+  T_{S,ratio} = \frac{29.44C+273.15C}{283.15C}
+\end{equation}
+
+where:
+
+$T_{WB,ratio} = $ ratio of load-side inlet air wet-bulb temperature in Kelvin to a reference temperature
+
+$T_{S,ratio} = $ ratio of source-side inlet water temperature in Kelvin to a reference temperature
+
+\begin{equation}
+	CoolPowCoeffModFac = CPC1 + CPC2 \left(T_{WB,ratio} \right)+CPC3 \left(T_{S,ratio} \right)+CPC4+CPC5
+\end{equation}
+
+where:
+
+CPC1 = user input for Cooling Power Consumption Coefficient 1
+
+CPC2 = user input for Cooling Power Consumption Coefficient 2
+
+CPC3 = user input for Cooling Power Consumption Coefficient 3
+
+CPC4 = user input for Cooling Power Consumption Coefficient 4
+
+CPC5 = user input for Cooling Power Consumption Coefficient 5
+
+
+The 4\(^{th}\) and 5\(^{th}\) coefficient (CPC4 and CPC5) used in the above equation are multipliers for the load-side and source-side flow ratios, respectively. For sizing, these ratios are assumed to be 1.
```

Comment:
I see you've followed the pattern of the other two sets of coefficients above, but all of these should include the VLratio and VSratio terms in the equations (even if the term is assumed to be 1 for sizing). These same equations are used during the simulation if I understand correctly.